src/wam2layers/tracking/backtrack.py

```diff
@@ -61,6 +61,15 @@ def backtrack(
         + region * precip * s_lower / (s_upper + s_lower)
         - evap * s_track_relative_lower
     )
+    # TODO: find better way to deal with negative values
```

Comment:
remove line 64 as we have addressed this now?

Reply:
done